.editorconfig

```diff
@@ -1,20 +1,21 @@
+
 root = true
 
 [*]
 charset = utf-8
 end_of_line = lf
 
+
 [*.java]
 indent_style = tab
-tab_width = 4
-trim_trailing_whitespace = true
 insert_final_newline = true
-ij_continuation_indent_size = 8
-max_line_length = 120
+trim_trailing_whitespace = true
+max_line_length = 150
+
 
 [*.{xml, xsd, dtd}]
 max_line_length = off
 indent_style = tab
 trim_trailing_whitespace = true
 insert_final_newline = true
-indent_size = 4
+indent_size = 4
```

Comment:
Add a new line here? (just to be consistent with our config `insert_final_newline = true` :-) )